jboss-eap-cd-openshift-launch/added/launch/messaging.sh

```diff
@@ -329,13 +373,28 @@ function inject_brokers() {
   defaultJmsConnectionFactoryJndi="$DEFAULT_JMS_CONNECTION_FACTORY"
 
   if [ "${#brokers[@]}" -eq "0" ] ; then
+    echo XX1
```

Comment:
?

Reply:
Sorry, still need to do a cleanup pass for some debug stuff.